app/src/androidTest/java/com/android/voyageur/ui/trip/schedule/ByDayScreenTest.kt

```diff
@@ -4,16 +4,77 @@ import androidx.compose.ui.test.assertIsDisplayed
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
 import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.android.voyageur.model.activity.Activity
+import com.android.voyageur.model.activity.ActivityType
 import com.android.voyageur.model.trip.Trip
 import com.android.voyageur.ui.navigation.LIST_TOP_LEVEL_DESTINATION
 import com.android.voyageur.ui.navigation.NavigationActions
+import com.android.voyageur.ui.navigation.Screen
+import com.android.voyageur.ui.trip.activities.createTimestamp
+import java.time.LocalDate
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertTrue
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
 
 class ByDayScreenTest {
```

Comment:
While for this screen particularly I could argue that there is never going to be an invalid date (since the activities already have dates chosen through a calendar picker), I gave it some thought and I updated the function to catch potential exceptions, perhaps if we'll want to use it to format other dates not chosen through a picker. That being said, I added your test as well, so thanks for the suggestion!